README.md

```diff
@@ -67,14 +67,16 @@ There are some formatter config exposed through the Jupyter Lab Advanced Setting
 Add an extra entry to your keyboard shortcuts settings with something like
```

Comment:
Should also add a note about jupyterlab version, possibly something like the following
```suggestion
Assumeing you are using `jupyterlab>=1.0.0`,  you can add an extra entry to your keyboard shortcuts settings with something like
```

Reply:
There’s a typo but otherwise, it looks great!

Reply:
Ah yes 🙈